src/components/HomepageFeatures/index.tsx

```diff
@@ -28,6 +30,7 @@ const featureList: FeatureItem[] = [
   //   description: 'Code reviews are required before merging to main',
   // },
   {
+    image: 'img/Git.svg',
```

Comment:
Why do we have 2 different GitHub icons? See line 4. It also looks useful to add something to invert darker logos in dark mode.

<img width="279" alt="Screenshot 2024-10-03 at 16 19 33" src="https://github.com/user-attachments/assets/ff4d0ab1-0370-4b49-90e4-91fd96112cbf">

Reply:
I've inverted the githubicon and adjusted the others their colors